src/Inflector.php

```diff
@@ -296,7 +296,7 @@ final class Inflector
     /**
      * @param string[] $rules The rules for converting a word into its plural form.
      *
-     * @psalm-param array<string, string> $rules
+     * @psalm-param non-empty-array<non-empty-string, string> $rules
```

Comment:
Should we check error on non-empty in this method?

Reply:
I've just tried to fix psalm errors. If you know better way let's do it